### PR TITLE
Allreviews

### PR DESCRIPTION
**Pull request by EduardoAquinta**
- Completed task 4 so have functional reviews view component. 
- have done basic cards with hover but needs a lot of css work. 
-

### CODE REVIEW

File: src/Components/ReviewsList.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }, []);
+
+
+    if (isLoading) return <p>Loading Reviews... please wait...</p>
+    return (
+        <section id="reviewList">
+            {reviews.reviews.map((review) => {
+                return (
+                    <ol type="1" key={review.review_id}>
+                        <ReviewListing
```
Comment (emilyb93) [inline src/Components/ReviewsList.jsx:25]: Your \<ol> tag here is rendering \<ReviewListing> as a child of it, but ReviewListing returns an article tag which means the direct child of \<ol> would be an article tag. Ordered and Unordered lists should only ever have list items as their direct children. Feel free to wrap a \<li> tag around the article tag in review listing to keep it semantic, but think more carefully about this in future

File: src/Components/ReviewListing.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export default function ReviewListing({title, designer, owner, review_img_url, category, created_at, votes, comment_count}) {
+    return (
+        <article id="reviewList">
+            <ul className="card">
+                <li><img src={review_img_url} alt="none found" id="listing"/></li>
+                <li>{title}</li>
```
Comment (emilyb93) [inline src/Components/ReviewListing.jsx]: I would probably change the html of this, \<li> tags are meant to house other tags, {title} is going to be a piece of text and therefore should be wrapped in a p tag. Also it doesnt make much sense as these items aren't really part of a list, they are just elements and you are using a list to render. You should be using CSS to style these correctly over html tags.

**Review by emilyb93 — CHANGES_REQUESTED** [review]

File: src/Components/ReviewsList.jsx (hunk 1 above)
Comment (EduardoAquinta) [reply src/Components/ReviewsList.jsx:25]: Ok I have changed the code, and removed the <ol> from around ReviewListing, and I have put the <li> around the article tag in the ReviewListing component.

File: src/Components/ReviewListing.jsx (hunk 2 above)
Comment (EduardoAquinta) [reply src/Components/ReviewListing.jsx]: I have removed the list tags too.

**Pull request merged**